tests/Core/List/FluentAutocompleteTests.FluentAutocomplete_Keyboard-ArrowDown.verified.razor.html

```diff
@@ -1,5 +1,5 @@
 
-<div class=" fluent-autocomplete-multiselect" style="width: 100%;" b-hg72r5b4ox="">
+<div class=" fluent-autocomplete-multiselect item-selected" style="width: 100%;" b-hg72r5b4ox="">
```

Comment:
@dvoituron - Is this an acceptable modification to the logic and thus the unit test?  A new class is being added if the count of the `SelectedOptions > 0`.  This shouldn't break anyone since the main class name is un-changed.

Reply:
Nop. It's not acceptable to change all Unit Tests.
You can do the "inverse": you can add an attribute `single-selection` only for you feature. And adapt the CSS in this case= `.fluent-autocomplete-multiselect[single-selection] { ... }`

```xml
<div class="fluent-autocomplete-multiselect" single-selection> ... 
```